package.json

```
@@ -11,6 +11,7 @@
     "check": "pnpm install --frozen-lockfile && echo 'Ignore paths from lint-staged'",
     "test": "SKU_TELEMETRY=false SKU_DISABLE_CACHE=true OPEN_TAB=false jest --verbose",
     "test:sku-init": "SKU_TELEMETRY=false SKU_DISABLE_CACHE=true OPEN_TAB=false jest --verbose --config=jest.config.sku-init.js",
+    "test:update": "pnpm run test:sku-init && pnpm --filter @sku-fixtures/braid-design-system exec sku test -u",
```

I would leave `sku-init` out of this one as it's best to just run that on CI. Also this isn't updating the `test` script snapshots`.
```suggestion
    "test:update": "pnpm run test -u && pnpm --filter @sku-fixtures/braid-design-system exec sku test -u",
```